.travis.yml

```diff
@@ -1,27 +1,36 @@
-dist: trusty
+cache: bundler
+
 language: node_js
 node_js:
-  - 8
-  
+  - '10.2'
+
 os:
-  - osx
   - linux
+  - osx
+  - windows
```

Comment:
Windows is still not supported by travis, this needs to be disabled before we can merge it. https://travis-ci.community/t/current-known-issues-please-read-this-before-posting-a-new-topic/264